src/test/java/com/iexec/worker/chain/WorkerpoolAuthorizationServiceTests.java

```diff
@@ -88,8 +88,7 @@ void shouldFailToPutWorkerpoolAuthorizationWhenCantGetWorkerPoolOwner() {
 
     @Test
     void shouldFailToPutWorkerpoolAuthorizationIfChainTaskIdIsNullInWorkerpoolAuthorization() {
-        WorkerpoolAuthorization workerpoolAuthorization = getWorkerpoolAuthorization();
-        workerpoolAuthorization.setChainTaskId(null);
+        final WorkerpoolAuthorization workerpoolAuthorization = WorkerpoolAuthorization.builder().build();
```

Comment:
To force `null` chain task id, we need this
```suggestion
        final WorkerpoolAuthorization workerpoolAuthorization = WorkerpoolAuthorization.builder().chainTaskId(null).build();
```